packages/stark-ui/src/modules/message-pane/components/message-pane.component.spec.ts

```diff
@@ -205,27 +204,6 @@ describe("MessagePaneComponent", () => {
 		mockMessagePaneService.getAll.and.returnValue(messageCollection$);
 	});
 
-	describe("on initialization", () => {
-		beforeEach(() => {
-			hostFixture.detectChanges(); // trigger initial data binding
-		});
-
-		it("should set internal component properties", () => {
-			expect(hostFixture).toBeDefined();
-			expect(component).toBeDefined();
-
-			expect(component.logger).not.toBeNull();
-			expect(component.logger).toBeDefined();
-			expect(component.messagePaneService).not.toBeNull();
-			expect(component.messagePaneService).toBeDefined();
-		});
-
-		it("should have default inputs", () => {
```

Comment:
I think we should keep this unit test at least to be sure that the properties are initialized with the correct value.